addons/project/data/project_demo.xml

```diff
@@ -119,7 +119,6 @@
         <record id="project_task_5" model="project.task">
             <field name="planned_hours" eval="76.0"/>
             <field name="user_id" ref="base.user_admin"/>
-            <field name="priority">1</field>
```

Comment:
which one should it be here? 1 or 0 ?

Reply:
As noted in PR description, in cases where the value differs, I've left the last value, since that value is being used (["Room 1: Decoration" on runbot](http://584509-master-50155a.runbot17.odoo.com/web#id=17&action=658&model=project.task&view_type=form&cids=&menu_id=463)).

Reply:
>  As noted in PR description, in cases where the value differs, I've left the last value, since that value is being used currently (["Room 1: Decoration" on runbot](http://584509-master-50155a.runbot17.odoo.com/web#id=17&action=658&model=project.task&view_type=form&cids=&menu_id=463)).


I think Its good to Keep last one and Remove first one as Last ones are actually used for those records (and in that way we can be sure we are not changing any behaviour)